lib/channel/src/lib.rs

```diff
@@ -167,7 +169,7 @@
         self.receiver_events.notify(1);
     }
 
-    // notify all senders with an event listener
+    // notify all receivers with an event listener
```

Comment:
Maybe update the comment on this and the next two methods

Reply:
Thanks. I'll fix it.